core/src/apps/common/paths.py

```diff
@@ -338,6 +338,9 @@ def match(path: Bip32Path) -> bool:
 # https://github.com/trezor/trezor-firmware/issues/1749
 PATTERN_SEP5_LEDGER_LIVE_LEGACY = "m/44'/coin_type'/0'/account"
 
+PATTERN_CASA = "m/45'/[0-1000000]/[0-1000000]/[0-1]/[0-1000000]"
```

Comment:
any reason for not using the actual schema `"m/45'/coin_type/account/change/address_index"`?

Reply:
Adding anti-ransom limits. An attacker could derive a change address at a non-standard path -- assuming they have compromised the wallet software.

Happy to use the actual schema. Let me know what you think.

Reply:
I think you misunderstood me. Yes, I'm suggesting using the string `"m/45'/coin_type/account/change/address_index"` because `coin_type`, `account`, `change` and `address_index` are well defined aliases, in use on all the surrounding lines. My question is, is there any strong reason you made the limits explicit? If not, please use the aliases.

Reply:
Fixed! and updated the description